packages/sdk-manager/src/SdkManagerContext.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  EVMAssetTransfer,
+  Environment,
+  EvmFee,
+  Fungible,
+  Transfer
+} from '@buildwithsygma/sygma-sdk-core';
+import { consume, createContext, provide } from '@lit/context';
+import { LitElement, html } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import {
+  WalletManagerContext,
+  WalletManagerController
+} from '@builtwithsygma/sygmaprotocol-wallet-manager';
+import { BaseProvider, TransactionRequest } from '@ethersproject/providers';
+import { Signer } from 'ethers';
+import { UnsignedTransaction } from '@ethersproject/transactions';
+import { SdkManagerState, SdkManagerStatus } from './types';
+
+export const SdkManagerContext = createContext<SdkManagerState | undefined>(
+  'sdk-context'
+);
+
+export class SdkManager implements SdkManagerState {
```

Comment:
Could we move this `SdkManager` class to it's own file and have here only the context one? Also, just a nit: I think `SDKManager` could make more sense for the class

Reply:
Sorry @wainola, which class name do you mean?

Reply:
Since this is the context file, the manager could go to it's own `SDKManager` file